presto-parser/src/main/antlr4/io/prestosql/sql/parser/SqlBase.g4

```diff
@@ -460,6 +464,9 @@ frameBound
     | expression boundType=(PRECEDING | FOLLOWING)  #boundedFrame
     ;
 
+updateSource
+    : (expression | DEFAULT)
```

Comment:
`DEFAULT` will be interpreted as an identifier (and as a column named "default"), but it should be treated a special entity that has distinct semantics. We'll need to introduce a `Default` expression for this.

Since we're not handling this at all in the code, I would get rid of it for now and do it as a future enhancement.

Reply:
Removed.